R/plotCCA.R

```diff
@@ -0,0 +1,540 @@
+#' Plot RDA or CCA object
+#'
+#' \code{plotRDA} and \code{plotCCA} create an RDA/CCA plot starting from the
+#' output of \code{\link[mia:runCCA]{CCA and RDA}} functions, two common methods
+#' for supervised ordination of microbiome data.
+#'
+#' @param object a
+#'   \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-constructor]{TreeSummarizedExperiment}}
+#'   or a matrix of weights. The latter is returned as output from \code{\link[mia:runCCA]{calculateRDA}}.
+#' 
+#' @param dimred A string or integer scalar indicating the reduced dimension to
+#'   plot. This is the output of \code{\link[mia:runCCA]{runRDA}} and resides in
+#'   \code{reducedDim(tse, dimred)}.
+#' 
+#' @param ellipse_fill TRUE or FALSE, should ellipses be opacity filled.
+#'   (default: \code{ellipse_fill = TRUE})
+#'
+#' @param ellipse_alpha Number between 0 and 1 to adjust the opacity of ellipses.
+#'   (default: \code{ellipse_alpha = 0.2})
+#'
+#' @param ellipse_size Number specifying the size of ellipses.
+#'   (default: \code{ellipse_size = 0.1})
+#' 
+#' @param ellipse_linetype Discrete number specifying the style of ellipses
+#'   (default: \code{ellipse_linetype = 1})
+#' 
+#' @param vec_size Number specifying the size of vectors.
+#'   (default: \code{vec_size = 0.5})
+#' 
+#' @param vec_colour String specifying the colour of vectors.
+#'   (default: \code{vec_color = "black"})
+#' 
+#' @param vec_color Alias for `vec_colour`.
+#' 
+#' @param vec_linetype Discrete number specifying the style of vector lines.
+#'   (default: \code{vec_linetype = 1})
+#' 
+#' @param arrow_size Number specifying the size of arrows.
+#'   (defaults: \code{arrow_size = 0.25})
+#' 
+#' @param label_size Number specifying the size of text and labels.
+#'   (default: \code{label_size = 4})
```

Comment:
Sorry... one thing came to my mind. We are switching from `para_meter `to `para.meter` in miaverse. So these underscores should be changed to dots, I just remembered....

functionName
parameter.name

See issue: https://github.com/microbiome/mia/issues/405